### PR TITLE
Fixed Jesus On Blocks Other Than Liquid Source

### PR DESCRIPTION
**Pull request by etianl**
Jesus was murdering me when I was walking on any lava other than source blocks, by causing the character to dip into it. This smol change fixes that

### CODE REVIEW

**Comment by BDARmech** [comment]
\no?

**Comment by etianl** [comment]
Yes, if you attempt to use jesus on blocks other than the source no matter what setting you choose it will dip you in over and over. This change fixes that

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Jesus.java [hunk 1]
```diff
@@ -215,9 +215,9 @@ else if ((event.fluidState.getFluid() == Fluids.LAVA || event.fluidState.getFlui
     private void onFluidCollisionShape(CollisionShapeEvent event) {
         if (event.state.getFluidState().isEmpty()) return;
 
-        if (event.state.getFluidState().isOf(Fluids.WATER) && !mc.player.isTouchingWater() && waterShouldBeSolid()) {
+        if (event.state.getBlock() == Blocks.WATER && !mc.player.isTouchingWater() && waterShouldBeSolid()) {
```
Comment (BDARmech) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Jesus.java:218]:
```suggestion
        if (event.state.getBlock() == Blocks.WATER && !mc.player.isTouchingWater() && waterShouldBeSolid()) {
```
Is the fluid tag/function obsolete, will it not be used?

Comment (etianl) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Jesus.java:218]: No idea if it's obsolete but it doesn't work properly for the portions of liquid that are flowing other than the source block

Comment (etianl) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Jesus.java:218]: This is a video showing how it acts without the code change. (Broken)
https://github.com/MeteorDevelopment/meteor-client/assets/115842502/eb3c48ba-9197-4e3b-86db-fe7429483cda
This is how it acts, after that smol change. (Fixed)
https://github.com/MeteorDevelopment/meteor-client/assets/115842502/bbef313e-dd35-428a-82a0-1ada241227c6

**Comment by etianl** [comment]
"maybe just add water and lava in collisions ("
Yeah this is a bypass to make Jesus work, but all the options in Jesus do nothing to change that it's deadly on lava and annoying on water (not the source blocks, just the blocks that flow from them) without the fix I provided. With that smol change to the if statements it behaves how it used to again before the update to 1.20.1

**Comment by etianl** [comment]
I had to fix the codes again because my change was not accounting for the waterlogged blocks as being water blocks

**Pull request merged**